packages/block-library/src/paragraph/edit.js

```diff
@@ -108,6 +123,36 @@ function ParagraphBlock( {
 					</ToolsPanelItem>
 				</InspectorControls>
 			) }
+			{ ! content && (
+				<Popover
+					anchorRef={ paragraphElement }
+					animate={ false }
+					position="top right left"
+					focusOnMount={ false }
+					__unstableForcePosition
```

Comment:
Just a note, as we're currently reworking a few things around the `Popover` component:

- use the `placement` prop instead of `position`
- use the `anchor` prop instead of `anchorRef` (available after #43691 gets merged, hopefully next week)
- use the `flip` and `resize` props instead of `__unstableForcePosition`